composer.lock

```diff
@@ -466,28 +466,28 @@
         },
         {
             "name": "phpcompatibility/phpcompatibility-paragonie",
-            "version": "1.3.0",
+            "version": "1.3.1",
```

Comment:
While I don't object to upgrading the versions in composer.lock, maybe that should be a separate PR?

Reply:
Oh nice catch, I think I upgraded and committed by mistake without realizing what branch I was on. That was clumsy of me.